src/core_modules/capture-core/components/Pages/ViewEvent/ViewEventComponent/viewEvent.actions.js

```diff
@@ -4,6 +4,7 @@ import type { OrgUnit } from '@dhis2/rules-engine-javascript';
 
 export const actionTypes = {
     VIEW_EVENT_FROM_URL: 'ViewEventFromUrl',
+    CHANGE_EVENT_FROM_URL: 'viewEventFromUrl',
```

Comment:
To avoid confusion with the already existing `VIEW_EVENT_FROM_URL` key, does it make sense to use ``changeEventFromUrl'' instead? Something like this: `CHANGE_EVENT_FROM_URL: 'changeEventFromUrl'`
Thanks!

Reply:
Actually that entry ended up not being used at all, so I'll remove it.